<launch>
    <include file = "$(find localization)/launch/racecar_model.launch"/>


    <include file = "$(find zed_wrapper)/launch/zed.launch"/>

    <node pkg="final_challenge" type="pure_pursuit.py" name="Trajectory_follower">
		<param name="odom_topic" value="/pf/pose/odom"/>
		<param name="drive_topic" value="/vesc/high_level/ackermann_cmd_mux/input/nav_2"/>
		<param name="speed" value="0.5"/>
		<param name="lookahead" value="4"/>
		<param name="trajectory_follow" value="False"/>
	</node>
	<node pkg="final_challenge" type="lane_trajectory.py" name="Trajectory_planner" output="screen">
		<param name="image_topic" value="/zed/zed_node/rgb/image_rect_color"/>
	</node>
	<node pkg="wall_follower" type="safety_controller.py" name="safety_controller">
	</node>
</launch>
